### ISSUE TITLE
🐛 Bug - Lack of focus indicator for "Log In" button and unfocusable "Register" button

### ISSUE THREAD

**Issue by BeauTheBeau** [Valour-Software/Valour#1151]
**Describe the bug**
There is no clear visual indicator that the Log In button is focused, and the Register button can not be focused through `Tab` 

**To Reproduce**

1. Go to the Valour login page
2. `Tab` to "Log in"
   - The Log In button will be focused, but there is no visual indicator
3. `Tab` to "Register"
   - The Register button is not focused, it skips to the next focusable element

**Expected behavior**
The buttons are focusable and have a clear indicator that they are focused 

**Screenshots**
You can see my keyboard inputs at the bottom of the screen. I'm not sure how useful this actually is, but it might help :)
<video src="https://github.com/Valour-Software/Valour/assets/90942493/3c9ab009-857f-4b68-be9b-6bb9abed0da4"></video>

**System Information:**
 - OS: Arch (BTW) Linux
 - Valour Version: V0.2.5.0
 - Valour Platform: Web
     - Browser: Firefox 117.0.1 (64-bit)

**Additional context**
The Log In button is a `button` element, but the Register is just a `div`

**Comment by BeauTheBeau** [comment]
Same issue applies for
- The Toggles in the "Create a Channel" dialog and User Settings
![image](https://github.com/Valour-Software/Valour/assets/90942493/d246f9a9-9eb3-4f27-9bd8-3d34947dc46d)
- "Delete my Account", "Save changes", "Copy Link" and other buttons of the same style
- "Save" button in Planet Settings/Info
- The Subscription cards
![image](https://github.com/Valour-Software/Valour/assets/90942493/a41451f8-20c0-43fa-990b-c462c5be7b71)

Sorry I didn't catch this in the main issue, was a bit sleepy

**Comment by SpikeViper** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Can't tab-highlight buttons on register/login page](https://trello.com/c/6tPKXgnA/87-cant-tab-highlight-buttons-on-register-login-page)
